sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -602,23 +602,36 @@ case class Least(children: Seq[Expression]) extends Expression {
 
   override def doGenCode(ctx: CodegenContext, ev: ExprCode): ExprCode = {
     val evalChildren = children.map(_.genCode(ctx))
-    ctx.addMutableState(ctx.JAVA_BOOLEAN, ev.isNull)
-    ctx.addMutableState(ctx.javaType(dataType), ev.value)
-    def updateEval(eval: ExprCode): String = {
+    val isNull = ctx.freshName("leastTmpIsNull")
```

Comment:
`val leastTmpIsNull = ctx.freshName("leastTmpIsNull")`

Reply:
`isNull`, `ev.isNull` and `eval.isNull` looks too close.

Reply:
yes, but I think we can stay with `ev.isNull` and there is no need for this new variable.